src/prefect/server/api/server.py

```diff
@@ -370,7 +370,7 @@ def create_ui_static_subpath():
         replace_placeholder_string_in_files(
             prefect.__ui_static_subpath__,
             "/PREFECT_UI_SERVE_BASE_REPLACE_PLACEHOLDER",
-            base_url,
+            base_url if base_url != "/" else "",
```

Comment:
The issue here was if `base_url == "/"` exactly? 

is `base_url==""` also an issue?

Reply:
Yeah except `base_url` defaults to `/` because that's the default on the setting

Reply:
So empty `base_urls` always ended up as `/`